packages/syntax-core/src/RadioButton/RadioButton.tsx

```diff
@@ -0,0 +1,89 @@
+import React, { ReactElement, useId } from "react";
+import classnames from "classnames";
+
+import styles from "./RadioButton.module.css";
+import Typography from "../Typography/Typography";
+
+const typographySize = {
+  sm: 100,
+  md: 200,
+};
+
+/**
+ * Radio Button (Base) This is a single lonely radio button with accompanying text
+ */
+const RadioButton = ({
+  checked = false,
+  disabled = false,
+  label,
+  size = "md",
+  error = false,
+  onChange,
+}: {
+  /**
+   * @defaultValue false
+   * Whether or not the box has been clicked
+   */
+  checked: boolean;
+  /**
+   * @defaultValue false
+   * Whether to show error color schema
+   */
+  error?: boolean;
+  /**
+   * @defaultValue "md"
+   * Size of the components
+   */
+  size?: string;
+  /**
+   * The callback to be called when the button is clicked
+   */
+  onChange: React.ChangeEventHandler<HTMLInputElement>;
+  /**
+   * Always add a label tag for best accessibility practices
+   */
+  label: string;
+  /**
+   * @defaultValue false
+   * Whether or not the box is disabled
+   */
+  disabled?: boolean;
+}): ReactElement => {
+  const checkedStyles = classnames(styles.outer, styles[size], {
+    [styles.errorOuter]: error,
+  });
+  const uncheckedStyles = classnames(styles.background, styles[size], {
+    [styles.errorBackground]: error,
+  });
+
+  const id = useId();
```

Comment:
This isn't necessary. You only need id if you need to explicitly link label to input, but since input is a child of label, you don't need it

Reply:
ohhh got it. I removed it, thank you